src/main/java/Duke.java

```diff
@@ -1,10 +1,66 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static String EXIT_CMD = "bye";
+    public static String LIST_CMD = "list";
+    public static String DONE_CMD = "done";
+    public static String TODO_CMD = "todo";
+    public static String DEADLINE_CMD = "deadline";
+    public static String EVENT_CMD = "event";
```

Comment:
Perhaps these commands being grouped together could have a common prefix? Following [coding standards](https://se-education.org/guides/conventions/java/index.html):

> Associated constants should have a common prefix.

```suggestion
    public static final String CMD_EXIT = "bye";
    public static final String CMD_LIST = "list";
    public static final String CMD_DONE = "done";
    public static final String CMD_TODO = "todo";
    public static final String CMD_DEADLINE = "deadline";
    public static final String CMD_EVENT = "event";
```